table_types.py

```diff
@@ -5,13 +5,14 @@
 
 class Table:
     @staticmethod
-    def create_table(msg_type: str):
+    def create_table(msg_type: str, **kwargs):
         if msg_type == 'BATTERY_STATUS':
             return BatteryStatusTable()
         elif msg_type == 'HEARTBEAT':
             return HeartbeatTable()
+        elif msg_type == 'GPS_INPUT':
+            return GPSInputTable(**kwargs)
```

Comment:
An important assumption here about which argument names are passed to the create_table() method, because the constructor for GPSInputTable() wants two named arguments: verbose and hdop. Is that intentional or an oversight?

Reply:
This is a bit of a hack. The "tables" idea is still WIP. Perhaps a discussion topic today.

Reply:
I got rid of **kwargs, should be a bit cleaner now.